README.md

```diff
@@ -107,6 +107,7 @@ A few projects building on GPflow and demonstrating its usage are listed below.
 | Project | Description |
 | --- | --- |
 | [GPflowOpt](https://github.com/GPflow/GPflowOpt)       | Bayesian Optimization using GPflow. |
+| [Trieste](https://github.com/secondmind-labs/trieste)       | Bayesian optimization with TensorFlow, with out-of-the-box support for GPflow models. |
```

Comment:
```suggestion
| [Trieste](https://github.com/secondmind-labs/trieste)       | Bayesian optimization with TensorFlow, with out-of-the-box support for GPflow (2.x) models. |
```
(together with other suggestion above) to clarify which one you might want to use when ?